sdk/rules_daml/daml.bzl

```diff
@@ -138,7 +149,7 @@ def _daml_build_impl(ctx):
             cp_srcs = "\n".join([
                 make_cp_command(
                     src = src.path,
-                    dest = "$tmpdir/" + src.path,
+                    dest = "$tmpdir/" + daml_source + src.path.removeprefix(ctx.label.package + "/"),
```

Comment:
To maintain the existing source paths of `daml/Daml/Script/...`, we allow an explicit daml_source flag in daml_compile. which will be put in paths and in the daml.yaml (so it makes it into the ghc inserted locations in LF)

Reply:
We also remove the existing ctx.label.package prefixes, which was putting all daml files under their full bazel path in the dars.